src/fheroes2/dialog/dialog_selectitems.cpp

```diff
@@ -1000,10 +972,172 @@ int Dialog::selectHeroes( const int heroId /* = Heroes::UNKNOWN */ )
 
 int Dialog::selectHeroType( const int heroType )
 {
+    fheroes2::Display & display = fheroes2::Display::instance();
+    fheroes2::StandardWindow background( 520, 280, true, display );
```

Comment:
What about making this window a bit lower?

```suggestion
    fheroes2::StandardWindow background( 520, 250, true, display );
```

![изображение](https://github.com/ihhub/fheroes2/assets/113276641/d2b420b4-46f2-479f-8184-d39b001525ab)